packages/open-truss/src/configuration/engine-v1/RenderConfig.tsx

```diff
@@ -134,8 +134,10 @@ function validateComponentProps(
   return errors
 }
 
-function setWorkflowSession(id: string): void {
-  localStorage.setItem(`OT-Workflow:${id}`, JSON.stringify({}))
+function useSetWorkflowSession(id: string): void {
+  useEffect(() => {
+    localStorage.setItem(`OT-Workflow:${id}`, JSON.stringify({}))
+  }, [id])
```

Comment:
This fixes this and improves it: effects are only run on the client so we will have access to `localStorage` _and_ it prevents extra writes to local storage